lib/v2/screens/sign_up/add_phone_number/add_phone_number.dart

```diff
@@ -33,37 +35,43 @@ class _AddPhoneNumberState extends State<AddPhoneNumber> {
         body: BlocConsumer<SignupBloc, SignupState>(
```

Comment:
I dont think you need a BlocConsumer here since listener is not being use.

Reply:
I need to navigate to the home page from here.